tests/VocabularyConfigTest.php

```diff
@@ -17,6 +17,8 @@ protected function setUp(): void
         setlocale(LC_ALL, 'en_GB.utf8');
         $this->model = new Model('/../../tests/testconfig.ttl');
         $this->assertNotNull($this->model->getVocabulary('test')->getConfig()->getPluginRegister(), "The PluginRegister of the model was not initialized!");
+
```

Comment:
Unnecessary whitespace?

Reply:
Good call! The trace of me trying to setup a global error handler...